src/DocumentDB.ChangeFeedProcessor/LeaseManagement/PartitionedByLeasePkCollectionRequestOptionsFactory.cs

```diff
@@ -0,0 +1,29 @@
+﻿//----------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  Licensed under the MIT license.
+//----------------------------------------------------------------
+
+namespace Microsoft.Azure.Documents.ChangeFeedProcessor.LeaseManagement
+{
+    using System;
+    using Microsoft.Azure.Documents.ChangeFeedProcessor.PartitionManagement;
+    using Microsoft.Azure.Documents.Client;
+
+    /// <summary>
+    /// Used to create request options for partitioned lease collections, when partition key is defined as /leasepk.
```

Comment:
/leasepk: please don't forget to adjust this to latest version of what we decided.